htdocs/user/card.php

```diff
@@ -1165,6 +1171,10 @@
 	print '<input class="maxwidth200" type="text" name="job" value="'.GETPOST('job').'">';
 	print '</td></tr>';
 
+	// Default warehouse
```

Comment:
If ($conf->stock->enabled) is false, all addition done for default warehouse must be hidden.
Can you add this condition for users not using the stock module ?

Reply:
ok it's done